README.md

```diff
@@ -100,11 +100,11 @@ Additional requirements for building on Windows:
 
 To build the Electron clients, run:
 
-    npm run action src/electron/build
+    npm run action src/electron/build -- --platform=windows --buildMode=debug
 
 To run the Electron clients, run:
 
-    npm run action src/electron/start
+    npm run action src/electron/start -- --platform=windows --buildMode=debug
```

Comment:
Currently we are calling `src/electron/build` in `src/electron/start`, to me it is somehow duplicated. Why not just running the electron app in `src/electron/start` (suppose we have already run `src/electron/build`)? In this case, we do not need to pass any build flags into this script any more.

Reply:
I feel like detecting whether or not the user as already run `src/electron/build` is starting to get outside the scope of this pr